src/Umbraco.Core/Extensions/AssemblyExtensions.cs

```diff
@@ -42,19 +42,6 @@ public static string GetRootDirectorySafe(this Assembly executingAssembly)
         return _rootDir;
     }
 
-    /// <summary>
-    ///     Returns the file used to load the assembly
-    /// </summary>
-    /// <param name="assembly"></param>
-    /// <returns></returns>
-    public static FileInfo GetAssemblyFile(this Assembly assembly)
```

Comment:
I think as noted we can't just remove this one, and several others, as they are public.  So I've re-targeted the PR to `v16/dev` and I'll create a new one that obsoletes what's removed in this one.

Reply:
Sound sensible, thanks! 🙌🏻